packages/account-v2/src/modules/FinancialAssessment/__tests__/FinancialAssessmentForm.spec.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { useAccountStatus, useActiveTradingAccount, useFinancialAssessment, useIsEuRegion } from '@deriv/api-v2';
+import { useDevice } from '@deriv-com/ui';
+import { render, screen } from '@testing-library/react';
+import { FinancialAssessmentForm } from '../FinancialAssessmentForm';
+
+jest.mock('../../FinancialAssessmentFields', () => ({
+    FinancialAssessmentFields: () => <div>FinancialAssessmentFields</div>,
+}));
+jest.mock('../../TradingExperienceFields', () => ({
+    TradingExperienceFields: () => <div>TradingExperienceFields</div>,
+}));
+
+jest.mock('@deriv/api-v2', () => ({
+    useAccountStatus: jest.fn(),
+    useActiveTradingAccount: jest.fn(),
+    useFinancialAssessment: jest.fn(),
+    useIsEuRegion: jest.fn(),
+}));
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn(() => ({
+        isMobile: false,
+    })),
+}));
+
+jest.mock('react-router-dom', () => ({
+    useHistory: jest.fn(),
+}));
+
+describe('FinancialAssessmentForm', () => {
+    it('renders submitted page', () => {
+        (useDevice as jest.Mock).mockReturnValue({ isMobile: true });
+        (useAccountStatus as jest.Mock).mockReturnValue({ data: { is_authenticated: true } });
+        (useIsEuRegion as jest.Mock).mockReturnValue({ isEUCountry: false });
+        (useActiveTradingAccount as jest.Mock).mockReturnValue({
+            data: {
+                is_virtual: false,
+            },
+        });
+        (useFinancialAssessment as jest.Mock).mockReturnValue({
+            data: {},
+            error: undefined,
+            isLoading: false,
+            mutation: {
+                error: undefined,
+                isLoading: false,
+                isSuccess: true,
+            },
+            update: jest.fn(),
+        });
```

Comment:
can we move these mocked return values to `beforeEach` and set a default value? I think it's more cleaner that way and when the test case need some minor changes from the hooks, you just have to define the specific value in the test case